test/invalid.js

```diff
@@ -129,6 +129,14 @@ describe("Error object structure", () => {
     const { level } = validation[0];
     expect(level).toBe("warning");
   });
+  it("allows `instanceof WebIDLParseError`", () => {
```

Comment:
```suggestion

  it("allows `instanceof WebIDLParseError`", () => {
```

Oops, we accidentally removed the newline before this line 😲